Demo/Demo/SwiftUIComponents/SwiftUICardDemo.swift

```diff
@@ -69,6 +69,23 @@ struct SwiftUICardDemo: View {
                 )
                 .cornerRadius(10)
                 .disabled(!baseViewModel.isCardFormValid(cardNumber: cardNumberText, expirationDate: expirationDateText, cvv: cvvText))
+                Button("Vault Card without Purchase") {
+                    guard let card = baseViewModel.createCard(
```

Comment:
Should this just be a constant instead of a guard let? It looks like we just return if we can't construct a card which would be necessary to vault a card without purchase.

Reply:
It's because I pass the card into the baseViewModel.vaultCard that expects a non optional value for card.